lib/jormungandr/test/unit/Cardano/Wallet/Jormungandr/BinarySpec.hs

```diff
@@ -71,6 +78,15 @@ spec = do
                     , parentHeaderHash = Hash {getHash = "\216OY\rX\199\234\188.<O\\\244Y\211\210\254\224`i\216\DC3\167\132\139\154\216\161T\174\247\155"}
                     }
                 []
+    describe "singleAddressFromKey" $ do
+        let pub = "3$\195xi\193\"h\154\&5\145}\245:O\"\148\163\165/h^\ENQ\245\248\229;\135\231\234E/"
+        it "encodes (network <> tag <> key) correctly into an address" $
+            singleAddressFromKey (Proxy @'Mainnet) pub
+            `shouldBe`
+            Address (BS.pack [3] <> pub)
```

Comment:
I'm thinking we can these simple cases here, and test `keyToAddress` more extensively together comparing with `jcli`

Reply:
Where does the `3` comes from here?

Reply:
`let singleAddressOnMainnet = 3`